charts/elasticsearch-master/templates/statefulset.yaml

```diff
@@ -184,6 +126,19 @@ spec:
 
         {{- end }}
       initContainers:
+        - command:
+          - sh
+          - -c
+          - chown -R 1000:1000 /master
+          image: {{ .Values.busybox.image }}
+          imagePullPolicy: IfNotPresent
+          name: fix-permissions
+          securityContext:
+            privileged: true
+          volumeMounts:
+          - mountPath: /master
```

Comment:
This should be changed. The persistance should be set at the mountPath /usr/share/elasticsearch/data, otherwise the data path, where the indices are saved in Elasticsearch are not going to be persisted. More info can be found [here](https://github.com/elastic/helm-charts/blob/master/elasticsearch/templates/statefulset.yaml#L333)